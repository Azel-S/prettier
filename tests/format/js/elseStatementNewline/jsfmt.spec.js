run_spec(__dirname, ["babel", "flow", "typescript"], { elseStatementNewLine: true });