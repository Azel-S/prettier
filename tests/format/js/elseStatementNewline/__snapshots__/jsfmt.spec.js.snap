// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`elseStatementNewline.js - {"elseStatementNewLine":false} format 1`] = `
====================================options=====================================
elseStatementNewLine: false
parsers: ["babel", "flow", "typescript"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
if (cond) {
    code;
} else {
    code;
}

if (cond) {
    code;
    // comment
    if (cond) {
        code;
    } else { //comment here
        code
    }
}    else {
    code;
}
=====================================output=====================================
if (cond) {
  code;
} else {
  code;
}

if (cond) {
  code;
  // comment
  if (cond) {
    code;
  } else {
    //comment here
    code;
  }
} else {
  code;
}

================================================================================
`;

exports[`elseStatementNewline.js - {"elseStatementNewLine":true} format 1`] = `
====================================options=====================================
elseStatementNewLine: true
parsers: ["babel", "flow", "typescript"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
if (cond) {
    code;
} else {
    code;
}

if (cond) {
    code;
    // comment
    if (cond) {
        code;
    } else { //comment here
        code
    }
}    else {
    code;
}
=====================================output=====================================
if (cond) {
  code;
}
else {
  code;
}

if (cond) {
  code;
  // comment
  if (cond) {
    code;
  }
  else {
    //comment here
    code;
  }
}
else {
  code;
}

================================================================================
`;
